tests/acceptance/features/bootstrap/WebDavLockingContext.php

```diff
@@ -839,6 +885,34 @@ public function waitForCertainSecondsToExpireTheLock(int $time): void {
 		\sleep($time);
 	}
 
+	/**
+	 * @Then :count locks should be reported for file :file inside the space :space of user :user by the WebDAV API
```

Comment:
should be adequate? 
```suggestion
	 * @Then :count locks should be reported for file :file inside the space :space of user :user
```

Reply:
yes but we also already have step like `And 0 locks should be reported for file "textfile.txt" of user "Alice" by the WebDAV API` so I was just following the existing pattern. what do I do?

Reply:
if it's new step then I would like to go with the suggestion